### PR TITLE
Boost: Remove global state from UpgradeCTA component 

### PR DESCRIPTION
**Pull request by dilirity**
## Proposed changes:

* Moves global state from UpgradeCTA component to parent component (Modules).

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

pc9hqz-286-p2

## Does this pull request change what data or activity we track or use?

no

## Testing instructions:

* Make sure Upgrade sections work as expected and yearly pricing shows correctly.

### Critical CSS

CleanShot 2023-09-28 at 12 48 59

### Image Size Analysis

CleanShot 2023-09-28 at 12 47 38

### CODE REVIEW

**Review by donnchawp — DISMISSED** [review]
Works as expected. The price shows in both areas.

File: projects/plugins/boost/app/assets/src/js/pages/settings/elements/UpgradeCTA.svelte [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 	import { __, sprintf } from '@wordpress/i18n';
-	import config from '../../../stores/config';
 	import RightArrow from '../../../svg/right-arrow.svg';
 	import { recordBoostEvent } from '../../../utils/analytics';
 	import routerHistory from '../../../utils/router-history';
 
 	const { navigate } = routerHistory;
 
 	export let description = '';
+	export let yearlyPricing;
```
Comment (pyronaur) [inline projects/plugins/boost/app/assets/src/js/pages/settings/elements/UpgradeCTA.svelte]: Missing type

**Review by pyronaur — CHANGES_REQUESTED** [review]

File: projects/plugins/boost/app/assets/src/js/pages/settings/elements/UpgradeCTA.svelte (hunk 1 above)
Comment (dilirity) [reply projects/plugins/boost/app/assets/src/js/pages/settings/elements/UpgradeCTA.svelte]: Addressed in 245cafb.

**Review by pyronaur — APPROVED** [review]
👍

**Pull request merged**